tests/python-gpu/test_gpu_updaters.py

```diff
@@ -94,13 +91,11 @@ def test_with_empty_dmatrix(self):
         np.testing.assert_allclose(predictions, 0.5, 1e-6)
 
     @pytest.mark.mgpu
-    def test_specified_gpu_id_gpu_update(self):
-        variable_param = {'gpu_id': [1],
-                          'max_depth': [8],
-                          'max_leaves': [255, 4],
-                          'max_bin': [2, 64],
-                          'grow_policy': ['lossguide'],
-                          'tree_method': ['gpu_hist']}
-        for param in parameter_combinations(variable_param):
-            gpu_results = run_suite(param, select_datasets=datasets)
-            assert_results_non_increasing(gpu_results, 1e-2)
+    @given(tm.dataset_strategy)
+    @settings(deadline=None, max_examples=10)
+    def test_specified_gpu_id_gpu_update(self, dataset):
+        param = {'tree_method': 'gpu_hist', 'gpu_id': 1}
```

Comment:
Any way that we can make `gpu_id` part of the strategy?

Reply:
Done